Content.Client/Camera/CameraRecoilSystem.cs

```diff
@@ -18,9 +22,14 @@ private void OnCameraKick(CameraKickEvent ev)
 
     public override void KickCamera(EntityUid uid, Vector2 recoil, CameraRecoilComponent? component = null)
     {
+        if (_configManager.GetCVar(CCVars.ScreenShakeIntensity) == 0)
+            return;
```

Comment:
you should be caching this on cvar changed because this function is called quite frequently and getting a cvar value is not a cheap call.

Reply:
Done.